python/tests/Examples.py

```diff
@@ -13,6 +14,15 @@
 # Running these tests will output the examples to the console
 class Examples(unittest.TestCase):
 
+    test_dir_path = os.environ.get("DIR_PATH", None)
+    def new_file(self, g, test_name):
```

Comment:
You should move this into the pyValidator.sh script. This script should handle both creating and deleting this directory.